media/com_finder/js/finder.es6.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * @copyright  Copyright (C) 2005 - 2017 Open Source Matters, Inc. All rights reserved.
+ * @license    GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+Joomla = window.Joomla || {};
+
+((Joomla, document) => {
+  'use strict';
+
+  document.addEventListener('DOMContentLoaded', () => {
+    const searchwordNodes = document.querySelectorAll('.js-finder-search-query');
+    const searchwords = [].splice.call(searchwordNodes);
+    searchwords.forEach((searchword) => {
+      // If the current value equals the default value, clear it.
+      searchword.addEventListener('focus', (event) => {
+        if (event.target.value === Joomla.JText._('MOD_FINDER_SEARCH_VALUE')) {
+          event.target.value = ''; // eslint-disable-line no-param-reassign
+        }
+      });
+
+      // Handle the auto suggestion
+      if (Joomla.getOptions('finder-search')) {
+        // If the current value is empty, set the previous value.
+        searchword.addEventListener('keypress', (event) => {
+          if (event.target.value.length > 1) {
+            Joomla.request({
+              url: `${Joomla.getOptions('finder-search').url}&q=${event.target.value}`,
+              method: 'GET',
+              data: { q: event.target.value },
+              perform: true,
+              headers: { 'Content-Type': 'application/x-www-form-urlencoded' },
+              onSuccess(response, xhr) {
+                response = JSON.parse(response); // eslint-disable-line no-param-reassign
```

Comment:
please dont use eslint-disable

Reply:
is there any workaround here ? its like having an extra variable for this

Reply:
yes